colonel/models/models.py

```diff
@@ -65,13 +65,25 @@ def __str__(self):
     def get_identifier_for(self, model: Model) -> str:
         return self.name if model == self.owner else f"{self.name}@{self.owner.name}"
 
+    def __eq__(self, other):
+        if not isinstance(other, Port):
+            return False
+        other_port: Port = other
```

Comment:
Why do you need this??
Can't we assume that equality is just for things of the same type?